app/views/flight/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Your results</h1>
```

Comment:
I think this file and the controller might be old ones that we had already updated to be plural...